templates/consul_service.json.j2

```diff
@@ -2,7 +2,7 @@
     "service": {
         "name": "{{ consul_service_name }}",
         "id": "{{ consul_service_name }}-{{ metadata_id }}",
-        "tags": [{{ consul_service_tags | map("to_json") | join(', ') }}],
+        "tags": [{{ consul_service_tags | join(', ') }}],
```

Comment:
Not sure to understand why this change?

Reply:
The change was due for tests, sorry for pushing it